### DISCUSSION TITLE
Problem get_results_for_nodes_deformations

### DISCUSSION THREAD

**Discussion by vitaly-novikov** [Dlubal-Software/Dlubal_CSharp_Client#55] [Q&A] [answered]
Hi there!
If the numbering of nodes has gaps (for example, 11, 12, 14), then the function
`nodes_deformations_row[] nodeDeformations1 = model.get_results_for_nodes_deformations(case_object_types.E_OBJECT_TYPE_LOAD_CASE, 1, 0); //all nodes -> 0`
outputs values with padding (the value ends up in another node). RFEM 6.04.0012
Vitaly
![image](https://github.com/Dlubal-Software/Dlubal_CSharp_Client/assets/122982897/0633fe7d-3a26-45d1-ab7e-614b4240093f)
![image](https://github.com/Dlubal-Software/Dlubal_CSharp_Client/assets/122982897/cedb93db-6e57-4cfc-ac80-888fb8f0be63)

**Comment by Rebecca-Coding21** [comment, marked as answer]
Hi @vitaly-novikov,
the node number is not retrieved by webservice results at the moment. The numbers that you receive are just indexes. I'm sorry for any inconveniences. We are already working on adding the object numbers to the webservice response.
